community/modules/files/fsi-montecarlo-on-batch/README.md

```diff
@@ -0,0 +1,60 @@
+## Description
+
+Copy files to a target GCS bucket.
+
+Primarily used for FSI - MonteCarlo Tutorial **[fsi-montecarlo-on-batch-tutorial]**.
+
+[fsi-montecarlo-on-batch-tutorial]: ../docs/tutorials/fsi-montecarlo-on-batch/README.md
```

Comment:
I would note somewhere, maybe here, that this module was designed to consume the outputs of the relevent other modules (pubsub, big-query, bucket etc)

Also mention that it is expected that a user will interact with these files by mounting the bucket. At first glance I was like, "how will these files get on the vm".